dashboard/app/models/user.rb

```diff
@@ -442,6 +442,10 @@ def teacher?
     self.user_type == TYPE_TEACHER
   end
 
+  def student_of?(teacher)
+    followeds.find_by_user_id(teacher.id).present?
```

Comment:
`followeds` -- spelling intentional?  If so you may want to add a comment that it's intentional.